2-import.R

```diff
@@ -0,0 +1,320 @@
+##############################################
+#               Functions                    #
+##############################################
+
+
+########################
+# Retrieve the data associated to a sensor
+########################
+
+
+#'  Retrieve the data associated to a sensor from the API Telraam
+#'
+#' @param nom Character. Name of the sensor, comes from the vector listeNom
+#' @param date1 Date. Start date "aaaa-mm-jj"
+#' @param date2 Date. End date "aaaa-mm-jj"
+#'
+#' @return Data from the API, from date1 to date2. Date1 and date2 are included.
+#' @export
+#' @examples
+#' retrieve_sensor('Burel', ymd('2021-05-21'),ymd('2021-05-30'))
+retrieve_sensor <- function(nom,date1,date2){
+  # Initialization
+  result <- data.frame() 
+  date2 <- date2 + days(1) # so that date2 is included
+  idCapteur <- listeSegments[which(listeNom==nom)] # retrieve the identifier associated to the name of the sensor
+  dates <- seq_by_3_month(date1,date2) # for the iteration of the retrieving, because when we call the API, the period can not exceed 3 month for each call
```

Comment:
```suggestion
  dates <- seq_by_3_month(date1,date2) # for the iteration of the retrieving, because when we call the API, the period can not exceed 3 months for each call
```